registry.json

```diff
@@ -1600,5 +1600,20 @@
     "telegram": "Pekac9",                                            
     "operatorName": "Pekac",                                
     "github": "pekac"
-  }
+  },
+ {
+  "ticker": "ORO",                                                      
+  "poolId": "4d827f2056879bce6a78963e1feba4825d80dc8a0143db3165931a92", 
+  "social": {                                                           
+  	"facebook": "", 
+  	"twitter": "@OroPool",
+  	"website": "http://www.oroops.com/",     
+  	"reddit": "",  
+  	"instagram": "",
+  	"youtube": ""
+  },
+  "telegram": "oro_pool",                                            
```

Comment:
You can't use a channel name in the telegram field.  You need to provide us with a valid telegram username as voting is done using your telegram account.

Reply:
Updated to username